### PR TITLE
Add slice packer utility for sending slice arg to contract ABI

### PR DESCRIPTION
**Pull request by dustinxie**
Test plan: make minicluster 5 minutes can pass

### CODE REVIEW

File: tools/executiontester/blockchain/contract.go [hunk 1]
```diff
@@ -12,22 +12,19 @@ import (
...
+	"time"
 
 	"github.com/iotexproject/iotex-core/action"
 	"github.com/iotexproject/iotex-core/address"
-	"github.com/iotexproject/iotex-core/config"
 	"github.com/iotexproject/iotex-core/pkg/hash"
 	"github.com/iotexproject/iotex-core/pkg/keypair"
 	"github.com/iotexproject/iotex-core/protogen/iotexapi"
 	"github.com/iotexproject/iotex-core/protogen/iotextypes"
+	"github.com/pkg/errors"
```
Comment (Frankonly) [inline tools/executiontester/blockchain/contract.go]: format import

File: tools/executiontester/assetcontract/assetcontract.go [hunk 2]
```diff
@@ -9,90 +9,95 @@ import (
...
 
 	"github.com/iotexproject/iotex-core/config"
-	"github.com/iotexproject/iotex-core/protogen/iotextypes"
-	"github.com/iotexproject/iotex-core/testutil"
 	"github.com/iotexproject/iotex-core/tools/executiontester/blockchain"
 )
 
+const (
+	// ChainIP is the ip address of iotex api endpoint
+	ChainIP = "localhost"
```
Comment (yutongp) [inline tools/executiontester/assetcontract/assetcontract.go]: private?

File: tools/executiontester/blockchain/contract.go [hunk 3]
```diff
@@ -231,7 +240,34 @@ func (c *contract) Transact(data []byte, readOnly bool) (string, error) {
...
+	// check interval set to 500ms
+	d := time.Duration(time.Millisecond * 500)
+	t := time.NewTicker(d)
+	defer t.Stop()
+
+	var rec *iotextypes.Receipt
+	var err error
+	checkNum := 120 // check times out after 120 tries = 60s
+
+	for {
```
Comment (yutongp) [inline tools/executiontester/blockchain/contract.go]: use backoff library

Comment (dustinxie) [reply tools/executiontester/blockchain/contract.go]: nice, thx!

**Comment by raullenchai** [comment]
CI failed badly. Need to dig out the reason

**Pull request merged**